src/modules/username.rs

```diff
@@ -0,0 +1,49 @@
+use ansi_term::Color;
+use std::env;
+use std::process::Command;
+
+use super::{Context, Module};
+
+/// Creates a segment with the current user's username
+///
+/// Will display the Node.js version if any of the following criteria are met:
+///     - The current user isn't the same as the one that is logged in ($LOGNAME != $USER)
+///     - The current user is root (UID = 0)
+///     - The user is currently connected as an SSH session ($SSH_CONNECTION)
+pub fn segment(_context: &Context) -> Option<Module> {
+    let user = env::var("USER").unwrap_or("".to_string());
+    let logname = env::var("LOGNAME").unwrap_or("".to_string());
+    let ssh_connection = env::var("SSH_CONNECTION").unwrap_or("".to_string());
+    let uid = get_uid().unwrap_or(1000);
```

Comment:
I defaulted to uid(1000) instead of just unwrap, since I'm not what to do instead of just using uid(1000) when it fails to get the uid.

Reply:
also, is it ok to default the other `env::var` to empty strings?

Reply:
I think in this situation, it may make more sense to have them as `Option<String>` values rather than defaulting to empty strings.

```rust
// This will turn a `Result` into an `Option`
let user = env::var("USER").ok();
```